src/js/DataSource.js

```diff
@@ -133,6 +133,35 @@ DataSource.prototype = {
      */
     setData: function(arrayOfUniformObjects) {
         this.data = arrayOfUniformObjects;
+    },
+
+    replaceIndent: '____________________________________________________',
+
+    fixIndentForTableDisplay: function(string) {
+        var count = string.search(/\S/);
+        var end = string.substring(count);
+        var result = this.replaceIndent.substring(0, count) + end;
```

Comment:
#### from tips & tricks dept
This also works and is simpler (when efficiency not an issue):
```javascript
var result = Array(count + 1).join('_') + end;
```